Functions.Templates/Templates/HttpTrigger-Python/__init__.py

```diff
@@ -4,21 +4,19 @@
 
 
 def main(req: func.HttpRequest) -> func.HttpResponse:
-    logging.info('Python HTTP trigger function processed a request.')
+    logging.info("Python HTTP trigger function processed a request.")
 
-    name = req.params.get('name')
-    if not name:
-        try:
-            req_body = req.get_json()
-        except ValueError:
-            pass
-        else:
-            name = req_body.get('name')
+    try:
+        # Get value for `name` from parameters or JSON body
+        name = req.params.get("name") or req.get_json().get("name")
+    except ValueError:
+        pass

```

Comment:
`name` will be undefined in this path, whereas before it would have been `None`

Reply:
Should I instead say

``` python
name = req.params.get("name") or req.get_json().get("name") or None
```

Reply:
Or, nevermind as the error thrown will mean that is never set. I'll keep at it.

Reply:
The easy way out is to just write 

``` python 
name = None
try:
	name = req.params.get("name") or req.get_json().get("name")
except ValueError:
    pass
```

Reply:
Just seems sloppy to me.

Reply:
```suggestion
        name = None
```

This will ensure you've assigned something to `name`. Alternatively, you could put `name = None` _before_ the `try` statement, but this is probably neater since you need the `pass` here anyway.

Reply:
I like this change.